server/src/main/java/org/apache/iotdb/db/query/reader/chunkRelated/ChunkReaderIterator.java

```diff
@@ -34,12 +35,12 @@
  * are used here: <code>ChunkReaderWithoutFilter</code> and <code>ChunkReaderWithFilter</code>.
  * <p>
  */
-public class DiskChunkReader implements IPointReader, IBatchReader {
+public class ChunkReaderIterator implements IPointReader, IBatchReader {
```

Comment:
This name is confusing... it seems could iterate chunk reader

Reply:
rename to ChunkDataIterator